ui/partials/perf/graphsctrl.html

```diff
@@ -13,11 +13,15 @@
             </button>
             <table>
               <tr>
-                <td width="10px" class="graph-legend" style="padding: 4px; background-color: {{series.color}}"></td>
+                <td width="10px" class="graph-legend" style="padding: 4px; background-color: {{series.mycolor}}"></td>
```

Comment:
So if you follow my suggestion above, we'll be keying background-color off of whether the series is active. So something like:

    background-color: {{series.active ? series.color | "grey"}}

I'd also like to add a css class here conditional on the series not being active which turns the text and the border into a lighter shade of grey, to more clearly indicate that it's not active.